#!/usr/bin/env python
PACKAGE = "hydra_visualizer"

import dynamic_reconfigure.parameter_generator_catkin as dr_gen  # NOQA

gen = dr_gen.ParameterGenerator()
gen.add("layer_z_step", dr_gen.double_t, 0, "size of the distance between layers", 5.0, 0.0, 50.0)
gen.add("collapse_layers", dr_gen.bool_t, 0, "whether or not to show the layers independently", False)

exit(gen.generate(PACKAGE, PACKAGE, "Visualizer"))
